doc: update known issues for NCS v2.8.0 (#17997)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/known_issues.rst

> Amazon Sidewalk

Reviewer: Just adding this here, please feel free to move this text to the most relevant section

```suggestion
 .. rst-class:: v2-8-0

NCSDK-28152: TPIU Trace Signal Skew on the nRF54H20 SoC
    Segger TPIU tracing on the nRF54H20 DK encounters instability due to a minimal skew between clock and data signals.
    This skew causes unreliable trace performance, resulting in intermittent data capture.

  **Workaround:** Adjusting the TPIU trace delay on SEGGER's J-Trace Pro can improve signal stability, enhancing ETM capture reliability.
                  For more information on adjusting trace timing, refer to the SEGGER Wiki's "Adjusting Trace Timings and General Troubleshooting" section.
                  
Amazon Sidewalk 
```

Also, links.txt need a link to this page: https://wiki.segger.com/Adjusting_trace_timings_and_general_troubleshooting#Type_3:_Trace_clock_delayed

Reviewer: @shanthanordic Please check this text

Author: Added to Other issues section.

Reviewer 2: that is fine.

KRKNWK-18465: BUS fault on FSK during the FACTORY_RESET call

>   The Bluetooth LE link is deinitialized.

Reviewer 3: ```suggestion
  The *Bluetooth®* LE link is deinitialized.
```

>   However, a race condition might occur where an event added by the *Bluetooth* LE link is not removed.

Reviewer 3: ```suggestion
  However, a race condition might occur where an event added by the Bluetooth LE link is not removed.
```

Bluetooth LE
============

> .. rst-class:: v2-8-0 v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-4-4 v2-4-3

Reviewer 4: DRGN-23231 has been fixed, remove v2-8-0

  **Affected platforms:** nRF5340

.. rst-class:: v2-8-0 v2-7-0

DRGN-22930: The SoftDevice Controller may de-reference a ``NULL`` pointer
  This can occur when using the vendor-specific HCI command Zephyr Write TX Power Level for a connection.

> 

Reviewer 4: Duplicated. I have added this issue DRGN-22930 in softdevice controller paragraph. 
You could remove it here.

  **Workaround:** Either disable host flow control (:kconfig:option:`CONFIG_BT_HCI_ACL_FLOW_CONTROL`) or cherry-pick commits from the upstream: `Zephyr PR #65272 <https://github.com/zephyrproject-rtos/zephyr/pull/65272>`_.

.. rst-class:: v2-8-0 v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-1 v1-9-0 v1-8-0 v1-7-1

DRGN-23518: ACL reassembly may deadlock
  This can happen if  the value of :kconfig:option:`CONFIG_BT_BUF_ACL_RX_COUNT` Kconfig option is less than or equal to the value of :kconfig:option:`CONFIG_BT_MAX_CONN`.

  **Workaround:** Ensure that the value of :kconfig:option:`CONFIG_BT_BUF_ACL_RX_COUNT` is greater than the value of :kconfig:option:`CONFIG_BT_MAX_CONN`.

.. rst-class:: v2-8-0

DRGN-23511: Building of multilink Bluetooth applications fails
  This happens when the :kconfig:option:`CONFIG_BT_BUF_RX_COUNT` Kconfig option is not explicitly set.

  **Workaround:** Set the Kconfig value explicitly.

.. rst-class:: v2-8-0

NCSDK-29354: Bluetooth traffic stalls while writing or erasing flash

>   Using system workqueue for flash operations while Bluetooth is in use could result in Bluetooth hang or flash operation failures (timeout in MPSL flash synchronization).

Reviewer 5: Good for me :+1:

>   Using system workqueue for flash operations while Bluetooth is in use could result in Bluetooth hang or flash operation failures (timeout in MPSL flash synchronization).

Author: Sure.

  Using system workqueue for internal flash operations while Bluetooth is in use could result in Bluetooth hang or flash operation failures (timeout in MPSL flash synchronization).
  This happens because Bluetooth RX context waits for the connection TX notify that is done in the system workqueue context.

  **Affected platforms:** nRF52 Series, nRF54L15

  **Workaround:** Use a separate workqueue for connection TX notify processing (:kconfig:option:`CONFIG_BT_CONN_TX_NOTIFY_WQ`).

Bluetooth Mesh
==============

The issues in this section are related to the :ref:`ug_bt_mesh` protocol.

.. rst-class:: v2-8-0 v2-7-0

NCSDK-29893: Adding dynamic names to devices does not work
  The advertiser options to automatically add the name in the scan response and the advertising data is removed.
  The Mesh files ``proxy_srv.c`` and ``pb_gatt_srv.c`` were using ``BT_LE_ADV_OPT_USE_NAME`` that has been removed.

.. rst-class:: v2-8-0 v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0

NCSDK-28363: Local composition hash generator only considers the first portion of the Composition Data Page 0 (CDP0) when the size of CDP0 exceeds ``BT_TX_SDU_MAX``

>   This affects applications that use the Large Composition Data server together with the default DFU Metadata format enabled using the :kconfig:option`CONFIG_BT_MESH_DFU_METADATA` Kconfig option.

Reviewer 6: ```suggestion
  This affects applications that use the Large Composition Data server together with the default DFU Metadata format enabled using the :kconfig:option:`CONFIG_BT_MESH_DFU_METADATA` Kconfig option.
```

.. rst-class:: v2-5-2

DRGN-21390: The ``disconnected`` callback might not get called on nRF5340
  The Bluetooth host running on the nRF5340 application core might deadlock on disconnection.
  This is due to a recent bugfix in the SoftDevice Controller triggering a bug in the ATT queuing layer.

  **Workaround:** Either disable host flow control (:kconfig:option:`CONFIG_BT_HCI_ACL_FLOW_CONTROL`) or cherry-pick commits from the upstream: `Zephyr PR #65272 <https://github.com/zephyrproject-rtos/zephyr/pull/65272>`_.

> .. rst-class:: v2-8-0 v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-1 v1-9-0 v1-8-0 v1-7-1

Reviewer 4: Remove v2-8-0, this has been resolved

The issues in this section are related to the :ref:`ug_matter` protocol.

> 

Reviewer 7: Could you add one new known issue?
```suggestion
.. rst-class:: v2-8-0

KRKNWK-19388: The smart plug functionality of Matter Bridge application does not work with Apple Home application.
  The additional functionality of a smart plug that runs alongside the Matter Bridge functionality is not visible in the Apple Home application.
  In fact, it applies to every functionality reflected by the static Matter endpoint and run alongside dynamic endpoints represented by the Matter Bridge.
```

  **Workaround:** While building the Matter Lock or Matter Template sample with the :makevar:`FILE_SUFFIX` variable set to ``release``, set the :kconfig:option:`CONFIG_NCS_SAMPLE_MATTER_WATCHDOG` Kconfig option to ``n``.

> .. rst-class:: vv2-7-0

Reviewer 7: ```suggestion
.. rst-class:: v2-7-0
```

Author: Whoops! Thanks for the sharp eye.

The issues in this section are related to the :ref:`ug_thread` protocol.

.. rst-class:: v2-8-0

KRKNWK-19628: On nRF54H20, persistence of keys is not yet supported
  After rebooting the device, all persistent keys (Thread's network key) will be lost.

> 

Reviewer 8: Could you add it to Matter chapter as well?

Author: We do not usually have the same issue in several places.

Reviewer 8: > Maybe we could say that this issue also applies to Matter.

Sounds good

The issues in this section are related to the :ref:`ug_thread` protocol.

.. rst-class:: v2-8-0

KRKNWK-19628: On nRF54H20, persistence of keys is not yet supported
  After rebooting the device, all persistent keys (Thread's network key) will be lost.

>   This issue applies also to Matter.

Reviewer 6: We could add the affected platform here.

The issues in this section are related to the :ref:`ug_thread` protocol.

> .. rst-class:: v2-8-0 v2-7-0

Reviewer 8: KRKNWK-19376 is fixed for 2.8.0

The issues in this section are related to the :ref:`ug_zigbee` protocol.

> .. rst-class:: v2-8-0 v2-7-0

Reviewer 9: This one is fixed in v2.8.0.

The issues in this section are related to the :ref:`ug_wifi` protocol.

.. rst-class:: v2-8-0 v2-7-0

> SHEL-2878: WPA3 security mode not working

Reviewer 10: This is resolved in 2.8.0 - please remove.

>   WPA3 security mode is not working for nRF54L15 PDK and nRF7002 EB.

Reviewer 10: - NCSDK-29650 : Wi-Fi connection times are increased in `_ns` builds. Increased the connection timeout (10s -> 15s) to work around this.
- NCSDK-29649 : Flash firmware integrity check doesn't work due to MbedTLS dependencies. Due to this nRF70 patches cannot be validated and corrupted patches can manifest in nRF70 bootup failures
- NCSDK-29651 : 54H20 Legacy crypto not supported. PSA crypto is used by default which limits Wi-Fi security only up to WPA2.
- SHEL-3226 : In nRF70 driver page docs, link to doxygen API has been removed

The issues in this section are related to the :ref:`ug_wifi` protocol.

.. rst-class:: v2-8-0

> NCSDK-29650: Wi-Fi connection times are increased in ``_ns`` builds

Reviewer 6: We could add the affected platform  nRF7002 for these Wi-Fi issues?

.. note::
    nRF Desktop is also affected by the Bluetooth LE issue :ref:`NCSDK-19865 <ncsdk_19865>`.

> .. rst-class:: v2-8-0 v2-7-0

Author: Still valid for 2.8.0 said Kamil offline.

.. rst-class:: v2-8-0 v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0

OCT-2070: Detection issues with USB-C to USB-C connection
  Using USB-C to USB-C when connecting an nRF5340 Audio DK to PC is not correctly detected on some Windows systems.

  **Affected platforms:** nRF5340 Audio DK

.. rst-class:: v2-8-0 v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0

OCT-2154: USB audio interface does not work correctly on macOS
  The audio stream is intermittent on the headset side after connecting the gateway to a Mac computer and starting the audio stream.
  This issue occurs sporadically after building the nRF5340 Audio application with the default USB port as the audio source.

  **Affected platforms:** nRF5340 Audio DK

.. rst-class:: v2-8-0 v2-7-0 v2-6-2 v2-6-1 v2-6-0

OCT-2172: The headset volume is not stored persistently
  This means the volume will fall back to default level after a reset.

  **Affected platforms:** nRF5340 Audio DK

> .. rst-class:: v2-8-0 v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0

Reviewer 11: Remove 2.8.0 tag

The issues in this section are related to the :ref:`nrf53_audio_app` application.

.. rst-class:: v2-8-0

> OCT-3179: CIS headset, potential for not establishing presentation synchronisation lock in bideriectional mode
>   If the CIS gateway is reset shortly after the headset has established synchronisation lock, this issue may be triggered.

Reviewer 3: ```suggestion
OCT-3179: CIS headset, potential for not establishing presentation synchronization lock in bidirectional mode
  If the CIS gateway is reset shortly after the headset has established synchronization lock, this issue may be triggered.
```

  **Workaround:** To start the charging again, turn the nRF5340 Audio DK off and then on again.

.. rst-class:: v2-8-0 v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0

OCT-2539: Presentation delay may not work as expected under some configurations
  The data is not presented at the correct time.

  **Affected platforms:** nRF5340 Audio DK

> .. rst-class:: v2-8-0 v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0

Reviewer 11: Remove tag v2.8.0

  **Affected platforms:** nRF9161

> 

Reviewer 12: .. rst-class:: v2-8-0

NCSDK-30050: The :ref:`nrf_cloud_rest_fota` sample experimental SMP FOTA support crashes with a secure fault approximately 1 out of 5 times.

Cellular samples
================

.. rst-class:: v2-8-0

> NCSDK-30050: The :ref:`nrf_cloud_rest_fota` sample with experimental SMP FOTA support enabled crashes with a secure fault.

Reviewer 3: ```suggestion
NCSDK-30050: The :ref:`nrf_cloud_rest_fota` sample with experimental SMP FOTA support enabled crashes with a secure fault
```

Libraries for networking
========================

> .. rst-class:: v2-8-0 v2-7-0

Reviewer 13: ```suggestion
.. rst-class:: v2-7-0
```

fixed already in mentioned PR.

  **Workaround:** Apply the fix from `sdk-nrf PR #14474`_.

> .. rst-class:: v2-8-0 v2-7-0

Reviewer 13: ```suggestion
.. rst-class:: v2-7-0
```

fixed.

  **Workaround:** To fix the issue, disable either the :kconfig:option:`CONFIG_MCUBOOT_HW_DOWNGRADE_PREVENTION` or :kconfig:option:`CONFIG_BOOT_FIH_PROFILE_LOW` Kconfig option in MCUboot.

> 

Reviewer 14: @peknis One more item for MCUboot 

`KRKNWK-18776: MCUboot updates take a long time
  The current implementation of MCUboot does not optimally utilize RRAM, which causes longer update times.

  **Affected platforms:** nRF54L15 DK
`

The issues in this section are related to :ref:`app_dfu`.

.. rst-class:: v2-8-0

NCSDK-30161 DFU from external flash does not work

  **Affected platforms:** nRF54H20

.. rst-class:: v2-8-0

> NCSDK-28241: Failed to execute SUIT payload fetch

Reviewer 15: Seems that we can remove this issue from the list, because now sporadic issue has been replaced by "NCSDK-30161 DFU from external flash does not work"

Author: Okay, will do. But your previous comment is hardly something we want to say in the docs. Can you formulate something customer oriented?

The issues in this section are related to :ref:`app_dfu`.

.. rst-class:: v2-8-0

NCSDK-30161 Combination of :kconfig:option:`CONFIG_ASSERT`, :kconfig:option:`CONFIG_SOC_NRF54H20_GPD`, and external flash causes an assertion during boot time

>   Device firmware update from external flash to an nRF54H20 device does not work.

Reviewer 15: Please remove this line

The issues in this section are related to :ref:`nrfxlib:softdevice_controller`.
In addition to the known issues listed here, see also :ref:`softdevice_controller_limitations` for permanent limitations.

> 

Reviewer 4: .. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0
DRGN-22443: A rare assert when disabling a periodic advertising set with responses
  In some rare cases, the controller can assert when disabling a periodic advertising set with responses

.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0
DRGN-22441: Fixed an issue where the length byte of the HCI packet could be incorrect
  This could happen when the packet contained an LE BIG Sync Established event or LE BIG Complete event with status not equal to success.

.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0
DRGN-22652:  Assert when receiving on the S8 Coded PHY
  This could happen in a connection where link peer device is transmitting on  on S8 Coded PHY.

.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0
DRGN-22686: Missing truncated advertising report
  The extended scanner would not generate a truncated advertising report after the coexistence interface aborted the reception of an ``AUX_CHAIN_IND`` packet.

.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0
DRGN-22678: A very rare issue where the controller stopped generating advertising reports
  On nRF52 Series and nRF53 Series devices, this would happen at least one hour after the scanner started.
  On nRF54L and nRF54H Series devices, this would occur immediately after the scanner started. 
  It would only happen when one of the following applies:
    * There was another central-like scheduling activity running. Examples of roles with such activities are the ACL central, periodic advertiser, isochronous broadcaster and the CIS central.
      This activity was configured with an event length or event spacing equal or greater than the scan interval.
      This is typically only true for use cases where the application enables isochronous channels or uses very short scan windows.
    * The scanner was configured with scan window equal to scan interval (continuous scanning).
    * The central-like scheduling activity required less than 1 ms to complete at the point in time where the scanner started.

.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 
DRGN-22230: A rare issue where the scanner would be stuck in the synchronizing state after failing to receive an ``AUX_ADV_IND`` packet
  This could only happen when the corresponding ``ADV_EXT_IND`` packet contained a resolvable address, private address resolution is enabled, and the periodic advertising list is not used. 

.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0
DRGN-22705: The controller could generate the LE Advertising Set Terminated event one event sooner than expected
  This could only happen when all of the following apply:
    * a non-zero Max_Extended_Advertising_Events parameter was used in the LE Set Extended Advertising Enable command.
    * other ongoing activities in the controller prevented the first advertising event from taking place when the advertising set was created.

22930

.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0
DRGN-22879: The Central could fail to receive the last packet in an isochronous event
  This could only happen if the Connected Isochronous Stream Creation procedure was initiated by the host before the Encryption Start procedure completed.

.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 
DRGN-23002: An assert could happen when using the coexistence interface
  This could happen when any of the following controller activities were ongoing:
    * Isochronous Broadcaster
    * Connected Isochronous channel in the peripheral role
    * Periodic Sync with Responses

.. rst-class:: v2-7-0
DRGN-23291:  LE Power Control was not being used for CIS
  This could happen when the CIS was not the first CIS in the CIG.

.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0
DRGN-23204: The SoftDevice Controller in the peripheral role could terminate a connection due to a MIC failure during a valid encryption start procedure
  This could only happen if the ``LL_ENC_RSP`` packet was corrupted due to on-air interference.

.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0
DRGN-23586: The received unframed Isochronous SDUs were not reported to be a SDU interval apart
  This could happen when the ISO interval is greater than the SDU interval and multiple SDUs can be received in a single ISO interval.

.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0
DRGN-23693: Wrong sleep clock accuracy
The sleep clock accuracy communicated to the peer was too inaccurate if MPSL was initialized with a low frequency clock accuracy better than 20ppm

.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0
DRGN-22036: A rare issue in the controller that could lead to a bus fault
  This could only happen when all of the following conditions were met:
    * The host was too slow at pulling HCI events.
    * One or more HCI events had been masked in the controller.
    * The controller was raising ACL or ISO data to the host

Author: Added. And these are all fixed in 2.8.0, right?

Reviewer 11: Sorry that is a question for Dragoon, I'm just dealing with Octave issues.

Author: Asking @laxiLang as he wanted to add these issues.

  **Affected platforms:** nRF5340, Thingy:53

.. rst-class:: v2-8-0 v2-7-0 v2-6-2 v2-6-1 v2-6-0

DRGN-21293: The LE Read ISO TX Sync command is implemented according to the raised errata ES-23138
  In ES-23138, the return parameter ``TX_Time_Stamp`` is used as the SDU synchronization reference of the SDU previously scheduled for transmission.
  This differs from the Bluetooth Core Specification v5.4, which uses the CIG or BIG reference anchor point as the SDU synchronization reference.
  When the CIG or BIG is configured with an ``ISO_Interval`` that equals the ``SDU_Interval``, there is no difference between the CIG or BIG reference anchor point and the SDU synchronization reference.
  If several SDUs are transmitted during each ``ISO_Interval``, meaning that it is larger than the ``SDU_Interval``, our implementation of the LE Read ISO TX Sync command returns a unique SDU synchronization reference for each SDU.

> .. rst-class:: v2-8-0 v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0

Reviewer 16: DRGN-20444 has been fixed

The issues in this section are related to :ref:`nrfxlib:softdevice_controller`.
In addition to the known issues listed here, see also :ref:`softdevice_controller_limitations` for permanent limitations.

> .. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0

Reviewer 17: We found one more issue

```
.. rst-class:: v2-8-0

DRGN-23776: Sending CIS packets with invalid MIC
  When the CIS central running on 5340 is sending encrypted ISO packets, the MIC may be invalid.
```

The issues in this section are related to the TF-M implementation in the |NCS|.

.. rst-class:: v2-8-0

NCSDK-29095: Writing assets to RRAM using TF-M causes increased interrupt latencies

>   Writes to RRAM provoke increased latencies to even the highest-priority interrupts.

Reviewer 18: ```suggestion
  Writes to RRAM provoke increased latencies even for the highest-priority interrupts.
```

Author: Not so sure about this. Can you @mia-ko check with the security guys?

Reviewer 19: Approving this suggestion. :slightly_smiling_face:

The issues in this section are related to the TF-M implementation in the |NCS|.

> .. rst-class:: v2-8-0

Reviewer 19: Expand all the way until v1.6.0.

> NCSDK-29095: Writing assets to RRAM using TF-M causes increased interrupt latencies

Reviewer 19: ```suggestion
NCSDK-29095: Writing assets to NVM using TF-M causes increased interrupt latencies
```

>   Writes to RRAM provoke increased latencies even for the highest-priority interrupts.
>   When TF-M is used on the nRF54L15 DK (board target ``nrf54l15dk/nrf54l15/cpuapp/ns``), writes to RRAM made by the secure image can disturb the normal flow of operations happening on the non-secure image.
>   This includes the SoftDevice Controller.
>   Its asserts may be triggered when the interrupt latency is too high, which will result in fatal errors.
>   Writes to RRAM happen on the secure image when writing assets to the Internal Trusted Storage (ITS) and the Protected Storage (PS).

Reviewer 19: ```suggestion
  Writes to NVM provoke increased latencies even for the highest-priority interrupts.
  When TF-M is used (``*/ns`` board targets), writes to NVM made by the secure image can disturb the normal flow of operations happening on the non-secure image.
 
  This includes the SoftDevice Controller on the nRF54L15 DK.
  Its asserts may be triggered when the interrupt latency is too high, which will result in fatal errors.

  The interrupt latency increases are much higher on the nRF53 and nRF91 Series devices (~80ms) compared to the nRF54L15 (~100us).

  Writes to NVM happen on the secure image when writing assets to the Internal Trusted Storage (ITS) and the Protected Storage (PS).
```

>   **Affected platforms:** nRF54L15DK

Reviewer 19: ```suggestion
  **Affected platforms:** nRF54L15, nRF5340, nRF91*
```
